<?xml version="1.0"?>
<launch>

 <!-- small description about your node -->
 
 <!--<node pkg="my_package_name" type="my_node_name" name="my_node_name"
 respawn="false" output="screen" args="$(find cooperative_control)/config/my_arg_file.yaml"/>-->
 
 <node pkg="cooperative_control" type="cooperative_control_node" name="CooperativeControlNode" respawn="false" output="screen">
	<rosparam command="load" file="$(find cooperative_control)/config/config_cooperative_control.yaml"/>
</node>

</launch>
